e2e_test/compaction/ingest_rows.slt

```diff
@@ -0,0 +1,49 @@
+statement ok
+CREATE SOURCE person
+    (id INTEGER, name VARCHAR, "email_address" VARCHAR, "credit_card" VARCHAR, city VARCHAR, state VARCHAR, "date_time" TIMESTAMP)
+with (
+  connector = 'nexmark',
+  nexmark.table.type = 'Person',
+  nexmark.split.num = '12',
+  nexmark.min.event.gap.in.ns = '0'
+) ROW FORMAT JSON;
+
+statement ok
+CREATE SOURCE auction (id INTEGER, "item_name" VARCHAR, description VARCHAR, "initial_bid" INTEGER, reserve INTEGER, "date_time" TIMESTAMP, expires TIMESTAMP, seller INTEGER, category INTEGER)
+with (
+  connector = 'nexmark',
+  nexmark.table.type = 'Auction',
+  nexmark.split.num = '12',
+  nexmark.min.event.gap.in.ns = '0'
+) ROW FORMAT JSON;
+
+statement ok
+CREATE SOURCE bid (auction INTEGER, bidder INTEGER, price INTEGER, "date_time" TIMESTAMP)
+with (
+  connector = 'nexmark',
+  nexmark.table.type = 'Bid',
+  nexmark.split.num = '12',
+  nexmark.min.event.gap.in.ns = '0'
+) ROW FORMAT JSON;
+
+statement ok
+CREATE MATERIALIZED VIEW nexmark_q7 AS
```

Comment:
What about create multiple MV, so that we have more than one table in `StaticCompactionGroupId::MaterializedView`.

Reply:
Consider that bug you fixed recently, which is only triggered when there are more than one table in a compaction group.

Reply:
I am afraid that multiple mv will extend the total ingestion time (computation become complex). We can consider this after we deploy the single mv version.